backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/Modifier.kt

```diff
@@ -68,6 +72,41 @@ internal fun Modifier.clickable(enabled: Boolean = true, bounded: Boolean = true
     )
 }
 
+internal fun Modifier.selectable(
```

Comment:
Why do we need this? Isn't this functionality covered by selectable modifier from Foundation?

Reply:
The Foundation one doesn't have an `onCLickLabel`, which we needed. Maria asked about this in her comments here https://github.com/Skyscanner/backpack-android/pull/1766#discussion_r1375892492

Reply:
Hey @bvitaliyg , this has now been changed, could you please review again and let me know if I missed anything? Thanks!